Doc/library/datetime.rst

```diff
@@ -114,7 +114,7 @@ Available Types
    :noindex:
 
    An idealized time, independent of any particular day, assuming that every day
-   has exactly 24\*60\*60 seconds.  (There is no notion of "leap seconds" here.)
+   has exactly ``24 * 60 * 60`` seconds.  (There is no notion of "leap seconds" here.)
```

Comment:
Why does this need code formatting? The existing output looks fine (https://docs.python.org/3.13/library/datetime.html#available-types).